src/model/entity/CompanyInfo.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require_once 'src/model/entity/Address.php';
+
+class CompanyInfo
+{
+    private int $companyInfoId;
+    private string $companyName;
+    private string $companyDescription;
+    private string $logoUrl;
+    private Address $address;
```

Comment:
Why does the company need an address? Is it like the "headquarters/main address" where it is based even tho there are multiple venue in other cities?